tests/components/zwave_mqtt/test_binary_sensor.py

```diff
@@ -0,0 +1,39 @@
+"""Test Z-Wave Sensors."""
+from homeassistant.components.binary_sensor import DEVICE_CLASS_MOTION
+from homeassistant.const import ATTR_DEVICE_CLASS
+
+from .common import setup_zwave
+
+
+async def test_binary_sensor(hass, generic_data, binary_sensor_msg):
+    """Test setting up config entry."""
+    receive_msg = await setup_zwave(hass, fixture=generic_data)
+
+    # Test Legacy sensor (disabled by default)
+    registry = await hass.helpers.entity_registry.async_get_registry()
+    entity_id = "binary_sensor.trisensor_sensor"
+    state = hass.states.get(entity_id)
+    assert state is None
+    entry = registry.async_get(entity_id)
+    assert entry
+    assert entry.disabled
+    assert entry.disabled_by == "integration"
+
+    # Test enabling legacy entity
+    updated_entry = registry.async_update_entity(
+        entry.entity_id, **{"disabled_by": None}
+    )
+    assert updated_entry != entry
+    assert updated_entry.disabled is False
```

Comment:
Can we add a message received for the legacy entity too, so we cover those lines.

Reply:
Sure, hopefully that will get us to 95 ;-)

Reply:
done!